src/navigation/SignInStack.js

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {createStackNavigator} from '@react-navigation/stack';
 import Home from '../screens/Home';
+import userProfileScreen from '../screens/userProfile';
```

Comment:
What is the user Profile screen? Why is it different from the previous ProfileScreen